auth-api/src/auth_api/models/product_code.py

```diff
@@ -36,3 +37,8 @@ def find_by_code(cls, code):
     def get_all_products(cls):
         """Get all of the products codes."""
         return cls.query.all()
+
+    @classmethod
+    def find_by_type_code(cls, type_code: str) -> List[ProductCode]:
+        """Find an Org instance that matches the provided id and not in INACTIVE status."""
+        return cls.query.filter(ProductCode.type_code == type_code).all()
```

Comment:
The comment mentioned not "INACTIVE", missed this filter here.

Reply:
good catch! copy-paste error :) Fixed it